__tests__/auth/helperFunctions.test.tsx

```diff
@@ -11,40 +11,30 @@ describe("test login helper functions", () => {
 	let stateValue: any;
```

Comment:
It's just to mock the state of the component that's being tested. It's any because it should be able to handle different values without making the test fail, but its value is never checked and probably by mocking the setstate function different this variable could also disappear

Reply:
Okay, that's fine. I mean, there is a type of "any" for a reason. I just want to make sure that you are using it consciously. 😊